x-pack/plugins/infra/public/pages/metrics/hosts/components/table/column_header.tsx

```diff
@@ -21,31 +21,56 @@ interface Props {
 const SEARCH_BAR_OFFSET = 250;
 const ANCHOR_SPACING = 10;
 
+const findTableParentElement = (element: HTMLElement | null): HTMLElement | null => {
+  let currentElement = element;
+
+  while (currentElement && currentElement.className !== APP_WRAPPER_CLASS) {
+    currentElement = currentElement.parentElement as HTMLElement;
```

Comment:
Is the type casting here necessary? When I removed `as HTMLElement` I saw the same type so can we just have: 
```suggestion
    currentElement = currentElement.parentElement;
```
?

Reply:
Good one. it's not needed. I was previously using `offsetParent` and that had to be cast to `HTMLElement`. With `parentElement` it's no longer needed.